templates/windows/neutron-ovs-agent.conf

```diff
@@ -27,7 +27,13 @@ firewall_driver = openvswitch
 local_ip = {{ compute_local_ip }}
 tunnel_bridge = br-tun
 integration_bridge=br-int
-ovsdb_connection=unix:\ProgramData\openvswitch\db.sock
+# ovsdb_connection=unix:\ProgramData\openvswitch\db.sock
 tenant_network_type=vxlan
-of_interface = ovs-ofctl
-ovsdb_interface = vsctl
+# of_interface = ovs-ofctl
+# ovsdb_interface = vsctl
+ovsdb_connection = tcp:127.0.0.1:6640
+of_interface = native
+ovsdb_interface = native
+[oslo_reports]
```

Comment:
Whops, a ramas asta in sample-ul ce l-am trimis. E optional (defapt, a trebuit sa fac un mic hack in neutron sa pot folosi oslo reports pe Windows), am folosit doar ca sa vad unde hang-uie procesul.